Makefile

```diff
@@ -1078,8 +1078,18 @@ test-debug:
 	env LIBSTORAGE_DEBUG=true $(MAKE) test
 
 test-rbd:
-	DRIVERS=rbd $(MAKE) deps
-	DRIVERS=rbd $(MAKE) ./drivers/storage/rbd/tests/rbd.test
+	env DRIVERS=rbd $(MAKE) deps
```

Comment:
Hi @codenrhoden,

The reason I don't use the `env` command in some places is because of what it does. The `env` command [removes](http://unix.stackexchange.com/a/103474) defined aliases and shell functions on which the build may rely.